metrics/grafana/tiflash_summary.json

```diff
@@ -6171,6 +6171,74 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "cards": {
+            "cardPadding": null,
+            "cardRound": null
+          },
+          "color": {
+            "cardColor": "#b4ff00",
+            "colorScale": "sqrt",
+            "colorScheme": "interpolateOranges",
+            "exponent": 0.5,
+            "mode": "spectrum"
+          },
+          "dataFormat": "timeseries",
```

Comment:
Seems that we should use "Time series buckets". 

Check other heatmaps for example
https://github.com/pingcap/tiflash/blob/fd1294b3ede9865bab09af291e813b91e7d665f9/metrics/grafana/tiflash_summary.json#L4411-L4452

![image](https://user-images.githubusercontent.com/4865550/186118020-13013730-ec62-4ad5-b916-173e39eec111.png)

Reply:
OK, I will change it. I choose the current style because all data points is at the bottom of the picture when using "Time series buckets". May be it needs a better bucket range.

Reply:
Fixed.